yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtilSuite.scala

```diff
@@ -148,4 +151,25 @@ class YarnSparkHadoopUtilSuite extends FunSuite with Matchers with Logging {
     }
 
   }
+
+  test("test expandEnvironment result") {
+    val target = Environment.PWD
+    var expect = "$" + target
```

Comment:
I'd do this whole block like so:

    val expected = if ('$$' exists) ... else if (isWindows) ... else ...